monkey/tests/unit_tests/common/configuration/test_agent_configuration.py

```diff
@@ -68,6 +69,30 @@ def test_custom_pba_configuration_schema():
     assert config.windows_filename == WINDOWS_FILENAME
 
 
+def test_custom_pba_configuration_schema__empty_filename_allowed():
+    schema = CustomPBAConfigurationSchema()
+
+    empty_filename_configuration = CUSTOM_PBA_CONFIGURATION.copy()
+    empty_filename_configuration.update({"linux_filename": ""})
+
+    config = schema.load(empty_filename_configuration)
+
+    assert config.linux_command == LINUX_COMMAND
+    assert config.linux_filename == ""
+    assert config.windows_command == WINDOWS_COMMAND
+    assert config.windows_filename == WINDOWS_FILENAME
+
+
+def test_custom_pba_configuration_schema__invalid_filename():
+    schema = CustomPBAConfigurationSchema()
+
+    invalid_filename_configuration = CUSTOM_PBA_CONFIGURATION.copy()
+    invalid_filename_configuration["linux_filename"] = "???"
```

Comment:
'???' is a vailid filename in Linux.
![image](https://user-images.githubusercontent.com/19957806/180994948-c764da76-e991-477c-a679-1b677a657552.png)